app/src/main/java/com/github/se/cyrcle/ui/report/AdminScreen.kt

```diff
@@ -472,16 +470,13 @@ fun AdminScreen(
                                               style = MaterialTheme.typography.titleMedium)
                                         },
                                         text = {
-                                          val selectedObject =
-                                              reportedObjectViewModel.selectedObject
-                                                  .collectAsState()
-                                                  .value
+                                          reportedObjectViewModel.selectedObject.collectAsState()
```

Comment:
I am not quite sure why we keep this call to collect as state when we don't assign it to a variable?

Reply:
I will remove it in the next commit I will just wait for you to finish your review so that I can address all your comments without hindering you on your review.